web/themes/custom/nysenate_theme/src/patterns/components/nysenate-header/nysenate-header.twig

```diff
@@ -57,19 +52,20 @@
 
                 {% if is_admin or admin or frontpage or mcp or lc or student %}
                   <li class="c-login--admin">
-                    {% if admin or is_admin %}
+                    {% if 'administrator' in user.getroles %}
```

Comment:
This logic should not be held in the template.  There's probably a better way to do this.